web/docs/try/quickstart.md

```diff
@@ -4,22 +4,22 @@ sidebar_position: 1
 
 # Quickstart
 
-This guide illustrates how you can use VAST from the command line interface.
-We're assuming that you have [installed VAST](../setup/install/README.md) and
-that you have a `vast` binary in your path.
+This guide illustrates how you can use Tenzir from the command line interface.
+We're assuming that you have [installed Tenzir](../setup/install/README.md) and
+that you have a `tenzir` binary in your path.
 
-## Start a VAST node
+## Start a Tenzir node
 
-Let's spin up a VAST node:
+Let's spin up a Tenzir node:
 
 ```bash
-vast start
+tenzir start
 ```
 
 Let's connect to the node and check its version via the `status` command:
```

Comment:
The idiomatic way to check the version is now `tenzir 'remote version'`

Reply:
Noted this down for the semantic pass, thanks.

Reply:
While health-checking might be done with `remote version`, I think a `status` operator will be the right replacement.